<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255,255,255);</string>
  </property>
  <widget class="QWidget" name="gridWidget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>251</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="1" column="2">
     <widget class="QPushButton" name="num_6">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">

QPushButton{
color: rgb(3, 86, 161);

}
QPushButton:pressed {

	color: rgb(0, 0, 0);

}

QPushButton{
background-color: rgbA(3, 86, 161,30);
color: rgb(0, 0, 0);
    border-width: 2px;
    border-radius: 10px;
}

QPushButton:pressed {
background-color: rgbA(3, 86, 161,60);
	color: rgb(0, 0, 0);
    border-width: 2px;
    border-radius: 10px;

}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="num_5">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">

QPushButton{
background-color: rgbA(3, 86, 161,30);
color: rgb(0, 0, 0);
    border-width: 2px;
    border-radius: 10px;
}

QPushButton:pressed {
background-color: rgbA(3, 86, 161,60);
	color: rgb(0, 0, 0);
    border-width: 2px;
    border-radius: 10px;

}</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="num_0">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">

QPushButton{
background-color: rgbA(3, 86, 161,30);
color: rgb(0, 0, 0);
    border-width: 2px;
    border-radius: 10px;
}

QPushButton:pressed {
background-color: rgbA(3, 86, 161,60);
	color: rgb(0, 0, 0);
    border-width: 2px;
    border-radius: 10px;

}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="num_enter">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">

QPushButton{
background-color: rgbA(3, 86, 161,30);
color: rgb(0, 0, 0);
    border-width: 2px;
    border-radius: 10px;
}

QPushButton:pressed {
background-color: rgbA(3, 86, 161,60);
	color: rgb(0, 0, 0);
    border-width: 2px;
    border-radius: 10px;

}</string>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="num_del">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">

QPushButton{
background-color: rgbA(3, 86, 161,30);
color: rgb(0, 0, 0);
    border-width: 2px;
    border-radius: 10px;
}

QPushButton:pressed {
background-color: rgbA(3, 86, 161,60);
	color: rgb(0, 0, 0);
    border-width: 2px;
    border-radius: 10px;

}</string>
      </property>
      <property name="text">
       <string>Del</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="num_9">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">

QPushButton{
background-color: rgbA(3, 86, 161,30);
color: rgb(0, 0, 0);
    border-width: 2px;
    border-radius: 10px;
}

QPushButton:pressed {
background-color: rgbA(3, 86, 161,60);
	color: rgb(0, 0, 0);
    border-width: 2px;
    border-radius: 10px;

}</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="num_8">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">

QPushButton{
background-color: rgbA(3, 86, 161,30);
color: rgb(0, 0, 0);
    border-width: 2px;
    border-radius: 10px;
}

QPushButton:pressed {
background-color: rgbA(3, 86, 161,60);
	color: rgb(0, 0, 0);
    border-width: 2px;
    border-radius: 10px;

}</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="num_7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">

QPushButton{
background-color: rgbA(3, 86, 161,30);
color: rgb(0, 0, 0);
    border-width: 2px;
    border-radius: 10px;
}

QPushButton:pressed {
background-color: rgbA(3, 86, 161,60);
	color: rgb(0, 0, 0);
    border-width: 2px;
    border-radius: 10px;

}</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="num_1">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">

QPushButton{
background-color: rgbA(3, 86, 161,30);
color: rgb(0, 0, 0);
    border-width: 2px;
    border-radius: 10px;
}

QPushButton:pressed {
background-color: rgbA(3, 86, 161,60);
	color: rgb(0, 0, 0);
    border-width: 2px;
    border-radius: 10px;

}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="num_3">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">

QPushButton{
background-color: rgbA(3, 86, 161,30);
color: rgb(0, 0, 0);
    border-width: 2px;
    border-radius: 10px;
}

QPushButton:pressed {
background-color: rgbA(3, 86, 161,60);
	color: rgb(0, 0, 0);
    border-width: 2px;
    border-radius: 10px;

}</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="num_2">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">

QPushButton{
background-color: rgbA(3, 86, 161,30);
color: rgb(0, 0, 0);
    border-width: 2px;
    border-radius: 10px;
}

QPushButton:pressed {
background-color: rgbA(3, 86, 161,60);
	color: rgb(0, 0, 0);
    border-width: 2px;
    border-radius: 10px;

}</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="num_4">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">

QPushButton{
background-color: rgbA(3, 86, 161,30);
color: rgb(0, 0, 0);
    border-width: 2px;
    border-radius: 10px;
}

QPushButton:pressed {
background-color: rgbA(3, 86, 161,60);
	color: rgb(0, 0, 0);
    border-width: 2px;
    border-radius: 10px;

}</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="keypad_val">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
